example/memcached-operator/memcached_controller.go.tmpl

```diff
@@ -95,11 +97,11 @@ func (r *ReconcileMemcached) Reconcile(request reconcile.Request) (reconcile.Res
 			// Request object not found, could have been deleted after reconcile request.
 			// Owned objects are automatically garbage collected. For additional cleanup logic use finalizers.
 			// Return and don't requeue
-			log.Printf("Memcached %s/%s not found. Ignoring since object must be deleted\n", request.Namespace, request.Name)
+			log.Error(err, "Memcached resource not found. Ignoring since object must be deleted", "Request.Namespace", request.Namespace, "Requst.Name", request.Name)
```

Comment:
Should this be an Error or a different log level? Error implies that something went wrong, but the `IsNotFound` situation is normal when a CR is deleted.